pipeline/tests/powdr_std.rs

```diff
@@ -381,3 +381,26 @@ fn btree() {
     let f = "std/btree_test.asm";
     execute_test_file(f, Default::default(), vec![]).unwrap();
 }
+
+mod reparse {
+
+    use powdr_pipeline::test_util::run_reparse_test_with_blacklist;
+    use test_log::test;
+
+    /// For convenience, all re-parsing tests run with the Goldilocks field,
+    /// but these tests panic if the field is too small. This is *probably*
+    /// fine, because all of these tests have a similar variant that does
+    /// run on Goldilocks.
+    const BLACKLIST: [&str; 5] = [
```

Comment:
Why these?

Reply:
They test machines designed to run on BN, so an assertion fails when run on GL.